### ISSUE TITLE
Accessibility of HexagonGrid

### ISSUE THREAD

**Issue by Nikomus** [neontribe/www#267]
**Describe the bug**
The HexagonGrid component traps screenreaders in a succession of missing image description alerts. This component should also be pausable, without the user have to maintain a pointer position to do so.

**To Reproduce**
1. Go to '/what-we-are-doing/'
2. Use a screenreader to read the page

**Expected behavior**
The component should have an accessible description. The animation should be clearly stoppable.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Screenreader: NVDA

**Comment by roseb04** [comment]
Needs to be split into 2 cards:
1. Hotfix accessibility sweep
2. UX review

**Comment by falconx** [comment]
@Nikomus 

> The HexagonGrid component traps screenreaders in a succession of missing image description alerts.

I wasn't able to reproduce this.

> This component should also be pausable, without the user have to maintain a pointer position to do so.

Where does this requirement come from? Is it mentioned in WCAG?

**Comment by Nikomus** [comment]
@falconx After updating NVDA I now can't reproduce the successive missing image descriptions - the section is just skipped. I think this component is meant to address user story:
"I want to see that Neontribe have worked with funding bodies before so that I can strengthen my funding application" so perhaps we should add some clarity to that alongside any accessibility improvements?

I think WCAG criterion 2.2.2 applies here:
https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html

**Comment by falconx** [comment]
>  perhaps we should add some clarity to that
Do you mean, making it clearer that the images refer to funding bodies?

> WCAG criterion 2.2.2
:ok_hand: I remember seeing this before now. couldn't remember if it was AAA or that was a threshold whether it was only necessary at a certain animation speed. But you're quite right, we should really have a play/pause button.

**Comment by Nikomus** [comment]
Yeah, I think if you're someone interested in knowing which funding bodies we've worked with you're more likely to be aware of what the logos mean, but in future we could have a little more information, and be screen-readable.

We may also want to look at this for the scrolling, but probs wait until we know any other changes
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

**Comment by falconx** [comment]
> We may also want to look at this for the scrolling, but probs wait until we know any other changes
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

I agree. I was thinking about this too. It's actually part of the CSS reset I now use on all my projects:
* https://github.com/falconx/components#css-reset

**Comment by falconx** [comment]
> we should really have a play/pause button.
I was thinking that a nice way to implement this could be like how we have "skip to content" links, where the link only appears on tab focus.

**Comment by katjam** [comment]
Discussed at #352:

Agreed that there is a user story about it being obvious that these projects are funded - rather than paid for by the organisation we build them for.

The animation and hexagons might not be the right solution for that. Considering the accessibility issue we thing a temp solution could be remove them and add a "funded by" line of copy to the individual project blocks.

We want some action to be taken but need to discuss with a wider group - and go back to the user story.

**Comment by Nikomus** [comment]
This isn't in the site anymore.
